### PR TITLE
Update jupyterHub to talk to EGO instead of DACO.

### PR DESCRIPTION
**Pull request by KevinFHartmann**

### CODE REVIEW

File: ansible/tasks/run_local.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@
     - "../vars/docker.yml"
     - "../vars/jupyterhub.yml"
     - "../vars/jupyter_notebook.yml"
-    - "../vars/oauth.yml"
-    - "../vars/daco.yml"
+    - "~/secrets/oauth_secrets.yml"
+    - "~/secrets/ego.yml"
```
Comment (KevinFHartmann) [inline ansible/tasks/run_local.yml]: Make this the same as for run_jukebox.yml
+    - "../vars/oauth.yml"
+    - "../vars/ego.yml"
+    - "{{ oauth_secrets }}"
+    - "{{ ego_secrets }}"

**Review by KevinFHartmann — COMMENTED** [review]
Great job, @jaserud !

File: ansible/tasks/run_local.yml (hunk 1 above)
Comment (jaserud) [reply ansible/tasks/run_local.yml]: Done

File: ansible/roles/jupyterhub/templates/V2_1__ego_setup.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+insert  into egoapplication (name, clientid, clientsecret, redirecturi, description, status, type) values ('{{ego_client_id}}','{{ego_client_id}}','{{ego_client_secret}}','{{ego_base_url}}','{{ego_client_id}}','APPROVED','ADMIN') on conflict do nothing;
+
+insert into policy (name) values ('{{ego_policy}}') on conflict do nothing;
+
+insert into egouser (name,email,type,firstname,lastname,createdat,lastlogin,status,preferredlanguage) values ('{{ego_user_email}}','{{ego_user_email}}','ADMIN','string','string',current_timestamp,current_timestamp,'APPROVED','ENGLISH') on conflict do nothing;
+
+insert into userpermission (user_id,policy_id,access_level) select u.id,p.id, 'READ' from egouser u, policy p where u.email = '{{ego_user_email}}' and p.name='{{ego_policy}}'  on conflict do nothing;
```
Comment (KevinFHartmann) [inline ansible/roles/jupyterhub/templates/V2_1__ego_setup.sql]: Everything else is customizable; the access level (READ/WRITE) should be, too.

File: ansible/roles/jukebox/templates/docker-compose.yml [hunk 3]
```diff
@@ -22,11 +22,14 @@ services:
       # Bind Docker volume on host for JupyterHub database and cookie secrets
```
Comment (KevinFHartmann) [inline ansible/roles/jukebox/templates/docker-compose.yml:22]: Let's make the location of the logs configurable, with a local default of "~/jukebox/logs".
"{{ log_directory }}:/dev/log:rw"

That way we can deploy it to production and make the system logs writeable by Jupyterhub, but still have it run locally on any machine, regardless of whether the user has a log device installed in their kernel (for instance, my Mac doesn't).

**Review by KevinFHartmann — COMMENTED** [review]
Very nice job!
A couple of tiny points, but otherwise, it's looking great!

File: ansible/roles/jukebox/templates/docker-compose.yml (hunk 3 above)
Comment (jaserud) [reply ansible/roles/jukebox/templates/docker-compose.yml:22]: done

File: ansible/roles/jupyterhub/templates/V2_1__ego_setup.sql (hunk 2 above)
Comment (jaserud) [reply ansible/roles/jupyterhub/templates/V2_1__ego_setup.sql]: and done

**Pull request merged**